.github/workflows/release-e2e.yaml

```
@@ -0,0 +1,54 @@
+---
+name: release-e2e
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  release-e2e-kind:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '~1.18'
+
+    - name: install_kind_binary
+      run: |
+        make kind
+
+    - name: start_kind_cluster
+      run: |
+        hack/tools/bin/kind create cluster
+        hack/tools/bin/kind export kubeconfig
```

With #384 these two jobs can be condensed to something like:
```suggestion
   - name: start_kind_cluster
      run: make kind-cluster
```

---

After that though, I'm good with this merging and will approve 👍🏻

---

Integrated your new actions. Also, reviewing that PR the UX improvements are great. Awesome job.